app/src/main/java/com/brewfind/Service/BreweryService.kt

```diff
@@ -9,7 +9,7 @@ import retrofit2.Callback
 import retrofit2.Response
 
 class BreweryService {
-    fun fetchBreweries() : MutableLiveData<ArrayList<Brewery>> {
+    internal fun fetchBreweries() : MutableLiveData<ArrayList<Brewery>> {
```

Comment:
Updated visibility modifiers to specify internal type
Source: https://kotlinlang.org/docs/visibility-modifiers.html